### PR TITLE
add home view and home controller js, add dummy data, add initial tes…

### PR DESCRIPTION
**Pull request by hsadler**
add home view and home controller js, add dummy data, add initial tests for HomeController, update app.js routing and index.html

### CODE REVIEW

File: test/client/HomeControllerSpec.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    createController = function() {
+      return $controller('HomeController', {
+        $scope: $scope
+      });
+    };
+
+    createController();
+  }));
+
+  // checking for test is just a test to test the tests
```
Comment (cassiebeckley) [inline test/client/HomeControllerSpec.js:24]: You really don't need to copy this line over; it's only relevant in the User controller tests.

**Comment by cassiebeckley** [comment]
Looks good! :+1:

**Comment by EricMFischer** [comment]
Nice!

**Comment by ntoscano** [comment]
+1
`

**Pull request merged**